content/en/docs/components/training/xgboost.md

```diff
@@ -0,0 +1,144 @@
++++
+title = "XGBoost Training"
+description = "Instructions for using XGBoost"
+weight = 30
+                    
++++
+
+{{% stable-status %}}
+
+This page describes XGBoostJob for training a machine learning model with XGBoost.
+
+## Installing Training Operator
+
+If you haven't already done so please follow the [Getting Started Guide](/docs/started/getting-started/) to deploy Kubeflow.
+
+
+## Verify that XGBoost support is included in your Kubeflow deployment
+
+Check that the XGboost custom resource is installed
+
+```
+kubectl get crd
+```
+
+The output should include `xgboostjobs.kubeflow.org`
+
+```
+NAME                                           AGE
+...
+xgboostjobs.kubeflow.org                       4d
+...
+```
+
+
+## Creating a XGBoost Job
+
+You can create XGBoost Job by defining a XGboostJob config file. See the manifests for the [IRIS example](https://github.com/kubeflow/tf-operator/blob/master/examples/xgboost/xgboostjob.yaml). You may change the config file based on your requirements. eg: add `CleanPodPolicy` in Spec to `None` to retain pods after job termination.
+
+```
+cat xgboostjob.yaml
+```
+Deploy the XGBoostJob resource to start training:
+
+```
+kubectl create -f xgboostjob.yaml
```

Comment:
@johnugeorge What do you think about using GitHub link here as I mentioned in this PR: https://github.com/kubeflow/website/pull/2920#issue-728146538
Then, users don't need to copy the example.